README.md

```diff
@@ -1,56 +1,66 @@
-# Distributed version of the Spring PetClinic Sample Application built with Spring Cloud 
+# Distributed version of the Spring PetClinic Sample Application built with Spring Cloud
 
-[![Build Status](https://github.com/spring-petclinic/spring-petclinic-microservices/actions/workflows/maven-build.yml/badge.svg)](https://github.com/spring-petclinic/spring-petclinic-microservices/actions/workflows/maven-build.yml)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Build Status](https://github.com/clemlesne/azure-spring-apps-workshop-081122/actions/workflows/maven-build.yml/badge.svg)](https://github.com/clemlesne/azure-spring-apps-workshop-081122/actions/workflows/maven-build.yml)
```

Comment:
Could you please rollback URL that reference your own repository?

Reply:
I'll fix it in a new commit.

Reply:
Fixed in 6418a633c5b30339e48187c2d87a319d6b19e67f